### PR TITLE
Issue 719 header param table schema doc

### PR DESCRIPTION
**Pull request by fflorent**
Document table-schema API.

### CODE REVIEW

File: api/grist.yml [hunk 1]
```diff
@@ -492,13 +493,41 @@ paths:
...
+            description: "Name of a table (normalized)."
+          required: true
+        - $ref: '#/components/parameters/headerQueryParam'
+      responses:
+        200:
+          description: A table's table-schema in JSON format.
+          content:
+            text/json:
+              schema:
+                type: string
```
Comment (paulfitz) [inline api/grist.yml:530]: This return type isn't very informative?

**Review by paulfitz — COMMENTED** [review]
Thanks @fflorent.

File: api/grist.yml (hunk 1 above)
Comment (paulfitz) [reply api/grist.yml:530]: Just to be clear on what I mean. The generated documentation can render examples of responses on the side. For instance:

Screenshot from 2023-11-15 14-41-54

Currently, this PR adds an endpoint whose response is rendered like this:

Screenshot from 2023-11-15 14-42-18

Comment (fflorent) [reply api/grist.yml:530]: We could probably use this table-schema: https://specs.frictionlessdata.io/schemas/table-schema.json

Comment (fflorent) [reply api/grist.yml]: Done!

File: api/grist.yml [hunk 2]
```diff
@@ -2074,6 +2103,37 @@ components:
...
+          type: string
+          enum: ["csv"]
+        mediatype:
+          type: string
+          enum: ["text/csv"]
+        encoding:
+          type: string
+          enum: ["utf-8"]
+        dialect:
+          $ref: https://specs.frictionlessdata.io/schemas/csv-dialect.json
```
Comment (fflorent) [inline api/grist.yml:2133]: If you feel uncomfortable with referencing external references, I can download the JSON and either include the YAML in the document or index as is locally in the repository. (also same remark for `schema`).

**Review by paulfitz — APPROVED** [review]
Great, thanks @fflorent !

**Pull request merged**